yarn.lock

```diff
@@ -2546,6 +2868,11 @@
   resolved "https://registry.yarnpkg.com/@nomiclabs/hardhat-ethers/-/hardhat-ethers-2.0.5.tgz#131b0da1b71680d5a01569f916ae878229d326d3"
   integrity sha512-A2gZAGB6kUvLx+kzM92HKuUF33F1FSe90L0TmkXkT2Hh0OKRpvWZURUSU2nghD2yC4DzfEZ3DftfeHGvZ2JTUw==
 
+"@nomiclabs/hardhat-ethers@^2.0.6":
```

Comment:
In the diff of yarn.lock I notice quite some duplication of packages with slightly different version.  
E.g. here 2.0.6 is added, but 2.0.5 kept in parallel. It's not clear to me why 2.0.5 wasn't replaced, imo it should have.